README.md

```diff
@@ -1,2 +1,55 @@
+**WARNING: this software is in pre-alpha. The happy flow works, but not all the flags combinations have been tested.**
+
+**BE CAREFUL USING IT WITH ALREADY EXISTING EBS VOLUME, DATA CAN BE LOST**
```

Comment:
What happens if you run it on an already existing volume?
Maybe you should specify it. I think the user expects that the directory will be copied on the existing volume.